### PR TITLE
Okx: Fix ping handler

### PR DESCRIPTION
**Pull request by gbjk**
Ping handler for okx is not the RFC compat version gorilla would support. Even if it was, I think the UseGorillaHandler is broken/misconceived because it does not send keepalive pings, it responds to them.

This also increases the delay to 27, in keeping with the 30 second timeout documented.

I think this issue was hidden by the noisy default subs. When just using funding-rates, it showed straight away.

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [x] Bug fix (non-breaking change which fixes an issue)

## How has this been tested

- [x] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gloriousCode — APPROVED** [review]
tACK!

```
[DEBUG] | WEBSOCKET | 17/05/2023 16:36:30 | Okx websocket connection: sending message [ping]
[DEBUG] | WEBSOCKET | 17/05/2023 16:36:31 | Okx websocket connection: message received: pong
```

File: exchanges/okx/okx_websocket.go [hunk 1]
```diff
@@ -224,9 +230,10 @@ func (ok *Okx) WsConnect() error {
 			ok.Websocket.GetWebsocketURL())
 	}
 	ok.Websocket.Conn.SetupPingHandler(stream.PingHandler{
-		UseGorillaHandler: true,
-		MessageType:       websocket.PingMessage,
-		Delay:             time.Second * 10,
+		UseGorillaHandler: false,
```
[the remark this answers is not in the document]
Comment (gbjk) [reply exchanges/okx/okx_websocket.go:233]: Yeah. I was going to do the same.

**Comment by gbjk** [comment]
Agreed. Back to you.

**Review by thrasher- — APPROVED** [review]
Thanks for identifying this and discussing it on Slack!

image

image

**Comment by gbjk** [comment]
Linter fixed. Will remember not to trust github "just commit this suggestion" in future.

**Pull request merged**